src/Pages/Login/Login.js

```diff
@@ -0,0 +1,152 @@
+import React, { Component } from "react";
+import "./Login.scss";
+
+class Login extends Component {
+  constructor() {
+    super();
+    this.state = {
+      email: "",
+      password: "",
+      pwpass: false,
+      errorActive: true,
+    };
+  }
+  loginEmail = (e) => {
+    this.setState({ email: e.target.value });
+    if (this.state.email.length > 5 && this.state.email.includes("@")) {
+      this.setState({ errorActive: true });
+    } else {
+      this.setState({ errorActive: false });
+    }
+  };
+
+  loginPassWord = (e) => {
+    this.setState({ password: e.target.value });
+    if (this.state.password.length >= 8) {
+      this.setState({ pwpass: true });
+    } else {
+      this.setState({ pwpass: false });
+    }
+  };
+
+  handleClick = () => {
+    fetch("http://10.58.4.149:8000/user/login", {
```

Comment:
```suggestion
    fetch(`${API}/user/login`, {
```
나중에 IP 주소가 바뀔 때를 대비하여 루트 폴더 아래 config.js라는 파일을 만들고 api 주소를 여기서 변수로 관리해주세요.

Reply:
네!